Vector2D.java

```diff
@@ -0,0 +1,66 @@
+package org.firstinspires.ftc.teamcode;
+
+/**
+ * Created by NoahR on 9/16/17.
+ */
+
+public class Vector2D {
+    private double m_nXComponent;
+    private double m_nYComponent;
+
+
+    public Vector2D( double iXComponent, double iYComponent ) {}
+
+    public double GetXComponent() { return m_nXComponent; }
+    public double GetYComponent() { return m_nYComponent; }
+
+    public double GetMagnitude()
+    {
+        return Math.sqrt( ( Math.pow(m_nXComponent, 2 ) ) + ( Math.pow(m_nYComponent, 2 ) ) );
+    }
+    public double GetDirection()
```

Comment:
This should be `Math.atan2` instead of `Math.atan`. `Math.atan` returns angles from `[-pi/2 pi/2]`, but `Math.atan2` returns angles from `[-pi, pi]`, and takes both the `x` and `y` components of the vector as arguments. If the vector was `<1, 1>`, the function in its current state would return `pi/4`. If it was `<-1, -1>`, it would also return `pi/4`, when it should return `-3pi/4` (or `5pi/4`)